projects/storefrontlib/cms-components/product/product-tabs/product-reviews/product-reviews.component.html

```diff
@@ -147,6 +166,16 @@ <h3>{{ 'productReview.overallRating' | cxTranslate }}</h3>
             *cxFeature="'!formErrorsDescriptiveMessages'"
             [control]="reviewForm.get('comment')"
           ></cx-form-errors>
+
+          <p
```

Comment:
Could you please consult with UI designers on the styling details of this component? You can tag them on the onboarding chat with links to the PR (and maybe ticket for context).

Reply:
From the points discussed with UI, I think these are the ones they made that are in scope:
2. lets do a limit of 300 and then introduce "read more"
3. the placing of the action buttons is usually on the right side as well :slightly_smiling_face: (not sure which action button they are talking about here though).